content/03.methods.md

```diff
@@ -197,43 +199,33 @@ We used RSEM [@doi:10/cwg8n5] v1.3.1 for both FPKM and TPM transcript- and gene-
-The package provides a unified interface for cell type quantification from RNA-seq gene expression data using multiple methods: EPIC (immune cells, fibroblasts, endothelial cells; n = 6), TIMER (immune cells; n = 6), MCP-counter (immune cells, fibroblasts, endothelial cells; n = 8), quanTIseq (immune cells; n = 10), CIBERSORT (immune cells; n = 22) and xCell (immune and non-immune cells; n = 64).
-To get a more fine grained deconvolution, we chose the top two most comprehensive methods i.e. xCell and CIBERSORT that are able to deconvolute 64 and 22 cell types, respectively.
-xCell tests for enrichment of cell types using ssGSEA and CIBERSORT uses ν-Support Vector Regression.
-xCell was our method of choice because 1) it is able to deconvolute the maximum number of immune and non-immune cell types, 2) is highly robust against background predictions and 3) can reliably identify the presence of immune cells at low abundances (0-1% infiltration depending on the immune cell type) [@pmid:31510660].
-Further, xCell was also shown to robustly outperform all other methods, including CIBERSORT in a benchmarking study [@pmid:29141660].
-We chose CIBERSORT as the second method because like xCell, it represents immune cell content as arbitrary scores that are highly correlated with actual cell proportions and are cross comparable between the two methods.
-The advantage of these methods is that they allow between samples (inter-sample), between cell types (intra-sample) and between cancer type (inter-histology) comparisons.
+#### Gene Set Variation Analysis
 
-To compare the outcome scores across the two profiling methods i.e. xCell and CIBERSORT, we took 13 cell types commonly represented between both methods and created a correlation plot per cell type per histology [@url:https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/immune-deconv/plots/corrplot_xCell_vs_CIBERSORT(abs.).png].
-Overall, across all histologies and cell types, the predictions did not correlate well (Pearson correlation = 0.12) but we did observe a high correlation (> 0.5) between certain cell type predictions i.e. Macrophages M2, Monocytes, Neutrophils and T cell CD8+ markers across specific histologies.
+We performed Gene Set Variation Analysis (GSVA) [@doi:10.1186/1471-2105-14-7] on collapsed, log2-transformed RSEM FPKM data using the GSVA bioconductor package [@doi:10.18129/B9.bioc.GSVA] and MSigDB hallmark gene sets [@doi:10.1016/j.cels.2015.12.004], setting `mx.diff=TRUE` to obtain Gaussian-distributed scores ([`gene-set-enrichment-analysis` analysis module](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/gene-set-enrichment-analysis)).
+We compared GSVA scores between histology groups (e.g., `short_histology`) using ANOVA and subsequent Tukey tests; p-values were Bonferroni-corrected for multiple hypothesis testing.
```

Comment:
> Also is short_histology an "e.g." or "i.e."?

Could go either way - I meant it in the "for example" sense, but could make the argument for this being an "in essence" situation.

Reply:
For context, we run it for `integrated_diagnosis` also. But that is not represented in the manuscript.

Reply:
> For context, we run it for `integrated_diagnosis` also. But that is not represented in the manuscript.

Okay, that is really what I was wondering. Since the manuscript only uses `short_histology`, we could probably just leave off the Latin for now.